_posts/2017-01-17-Oredev2016.md

```diff
@@ -0,0 +1,695 @@
+---
+layout: post
+authors: [kevin_van_houtte]
+title: 'Øredev 2016'
+image: /img/oredev/oredevonwhite.png
+tags: [Conference]
+category: Conferences
+comments: true
+---
+
+When arrived in Malmö city, we were welcomed with an exploration tour of the third biggest city in Sweden.
+The next day Øredev 2016 officially started!
+It was noticeable that there was a healthy combination for everyone: "Microservices", "Docker", "UX", "Management", etc...
+All these buzzwords were applied in various talks.
+This blog post will be about the talks I favored and where I learned the most from and still use in my nowadays projects.
+
+<p style="text-align: center;">
+  <img style="max-width:640px;" alt="Oredev begin" src="/img/oredev/oredevbegin.jpg">
+</p>
+
+# The Overview Wizard
+
+* [Dockerizing Microservices](#microservices)
+* [Secure your Docker](#docker)
+* [Best Practices & Traps to avoid in UX](#ux)
+* [Kubernetes](#kubernetes)
+* [Linked data API: The future of HTTP API's?](#api)
+
+
+<a name="microservices" />
+
+# Dockerizing Microservices
+
+## The Slideless Microservices session (Adam Bien)
+
+<img class="float-image p-image" src="{{ '/img/oredev/adam-bien.jpeg' | prepend: site.baseurl }}" alt="Adam Bien" style="width: 100px"/>
+
+[Adam Bien](https://twitter.com/adambien) is an Expert Group member
+ for the Java EE 6 and 7, EJB 3.X, JAX-RS, and JPA 2.X JSRs. He has worked with
+ Java technology since JDK 1.0 and with Servlets/EJB 1.0 and is now an architect and developer
+ for Java SE and Java EE projects. He has edited several books about JavaFX, J2EE, and Java EE,
+ and he is the author of Real World Java EE Patterns—Rethinking Best Practices and Real World Java
+ EE Night Hacks—Dissecting the Business Tier. Adam is also a Java Champion, Top Java Ambassador 2012,
+ and JavaOne 2009, 2011, 2012, 2013 and 2014 Rock Star.
+
+In this session Adam guided us in how to dockerize and communicate between microservices.
+He says that involving the container technology to the microservices eco system is considered the future and inevitable.
+When looking into the communication between dockerized microservices, there are several ways in doing so.
+Before communicating, we first make the service dockerized with the help of a Dockerfile.
+
+The Dockerfile will be the configuration for your image:
+
+{% highlight bash %}
+
+FROM openjdk:8-jre                     // Which base image do I run from?
+VOLUME /tmp                            // Is where you will send data to.
```

Comment:
I would leave out any periods in comments unless the comment consists of multiple sentences.